core/renderers/common/block_rendering.js

```diff
@@ -51,19 +48,6 @@ const {Theme} = goog.requireType('Blockly.Theme');
 const {TopRow} = goog.require('Blockly.blockRendering.TopRow');
 const {Types} = goog.require('Blockly.blockRendering.Types');
 
-/**
- * Returns whether the debugger is turned on.
- * @return {boolean} Whether the debugger is turned on.
- * @alias Blockly.blockRendering.isDebuggerEnabled
- * @package
- */
-const isDebuggerEnabled = function() {
-  deprecation.warn(
-      'Blockly.blockRendering.isDebuggerEnabled()', 'September 2021',
-      'September 2022', 'Blockly.blockRendering.debug.isDebuggerEnabled()');
```

Comment:
Are we OK removing this before the promised September 2022?

Reply:
I'm okay with removing it and changing these warnings to say where it went, then removing the warnings entirely after september 2022.

Reply:
Ok, I updated these messages as well as added back in `Blockly.blockRendering.debug` since those are what we recommended people switch to. Also just wanted to note that all these methods are technically @package and have been since they were created.